package.json

```diff
@@ -33,15 +33,24 @@
     "babel-cli": "^6.24.0",
     "babel-preset-env": "^1.2.2",
     "browser-sync": "^2.18.8",
+    "chai": "^3.5.0",
     "eslint": "^3.18.0",
     "eslint-config-airbnb-base": "^11.1.2",
     "eslint-plugin-import": "^2.2.0",
     "fs-extra": "^0.30.0",
+    "karma": "^1.5.0",
+    "karma-firefox-launcher": "^1.0.1",
+    "karma-mocha": "^1.3.0",
+    "karma-phantomjs-launcher": "^1.0.4",
```

Comment:
Sure we can drop phantom, I just thought we wanted to go with the headless approach.